src/main.rs

```diff
@@ -261,35 +264,131 @@ fn main() -> Result<(), StrErr> {
 	let mode = CompileMode::Check { test : false };
 	let compile_opts = args.compile_options(&config, mode, Some(&ws))?;
 
+	let metadata = {
+		let opts = OutputMetadataOptions {
+			features: vec![],
```

Comment:
Is there a way to somehow derive these options from the cli args? Because users should be able to specify features etc.

Reply:
Ok. fixed.